Accounts.sql

```diff
@@ -1,50 +1,10 @@
-insert into Accounts (Account_No, customer_ID, acc_status, acc_open_date, account_type, account_balance) values (35, 647646828, 'stable', '2021-09-07', 'savings', 4196);
-insert into Accounts (Account_No, customer_ID, acc_status, acc_open_date, account_type, account_balance) values (36, 487340279, 'stable', '2021-12-16', 'savings', 16610);
-insert into Accounts (Account_No, customer_ID, acc_status, acc_open_date, account_type, account_balance) values (37, 413013533, 'signing', '2021-06-02', 'salary', 19007);
-insert into Accounts (Account_No, customer_ID, acc_status, acc_open_date, account_type, account_balance) values (38, 725140382, 'blocked', '2021-02-27', 'savings', 8316);
-insert into Accounts (Account_No, customer_ID, acc_status, acc_open_date, account_type, account_balance) values (39, 487340279, 'stable', '2021-03-18', 'savings', 6651);
-insert into Accounts (Account_No, customer_ID, acc_status, acc_open_date, account_type, account_balance) values (40, 096160943, 'blocked', '2021-10-26', 'current', 8959);
-insert into Accounts (Account_No, customer_ID, acc_status, acc_open_date, account_type, account_balance) values (41, 444726060, 'blocked', '2021-05-31', 'salary', 17966);
-insert into Accounts (Account_No, customer_ID, acc_status, acc_open_date, account_type, account_balance) values (42, 295542389, 'closed', '2021-05-22', 'savings', 5134);
-insert into Accounts (Account_No, customer_ID, acc_status, acc_open_date, account_type, account_balance) values (43, 647646828, 'closed', '2021-07-07', 'savings', 15068);
-insert into Accounts (Account_No, customer_ID, acc_status, acc_open_date, account_type, account_balance) values (44, 657376957, 'closed', '2021-05-31', 'salary', 9679);
-insert into Accounts (Account_No, customer_ID, acc_status, acc_open_date, account_type, account_balance) values (45, 775630433, 'closed', '2021-12-30', 'savings', 11132);
-insert into Accounts (Account_No, customer_ID, acc_status, acc_open_date, account_type, account_balance) values (46, 657376957, 'stable', '2021-05-01', 'current', 10624);
-insert into Accounts (Account_No, customer_ID, acc_status, acc_open_date, account_type, account_balance) values (47, 091798503, 'stable', '2021-09-23', 'current', 13107);
-insert into Accounts (Account_No, customer_ID, acc_status, acc_open_date, account_type, account_balance) values (48, 105965720, 'blocked', '2021-09-23', 'salary', 6948);
-insert into Accounts (Account_No, customer_ID, acc_status, acc_open_date, account_type, account_balance) values (49, 176442237, 'closed', '2021-11-23', 'current', 15867);
-insert into Accounts (Account_No, customer_ID, acc_status, acc_open_date, account_type, account_balance) values (50, 444726060, 'signing', '2021-05-30', 'current', 3336);
+insert into Accounts (number, atm_pin, customer_id, status, type, balance) values (7957121942, 1676, 2, 'blocked', 'salary', 36572.71);
```

Comment:
- Need lot more data to see impact of index. 
- how do you deduce values of customer ids? Aren't they autogenerated? How is it guaranteed to generate those ids?
- You can batch insert rows using one sql statement.
- There is generate series function which can used to generate lot of data or refer this nodes script I used http://github.com/pradosh987/db_seed

Reply:
Okay sir will check.